src/runtime_src/core/tools/xbtracer/src/lib/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+link_directories($ENV{XILINX_XRT}/lib)
+
+set(SRCS
+  capture.cpp
+  logger.cpp
+  xrt_device_inst.cpp
+  xrt_kernel_inst.cpp
+  xrt_bo_inst.cpp
+  xrt_hw_context_inst.cpp
+  xrt_xclbin_inst.cpp
+)
+
+set (LIBS xrt_coreutil)
```

Comment:
My recommendation is to not use these sort of variables, but instead list the files explicitly in the targets.

Reply:
changed as recommended, please check the latest update